js/packages/teams-ai/src/prompts/AssistantMessage.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @module teams-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TemplateSection } from "./TemplateSection";
+import { TurnState } from '../TurnState';
+
+/**
+ * A message sent by the assistant.
+ */
+export class AssistantMessage<TState extends TurnState = TurnState> extends TemplateSection {
+    /**
+     * Creates a new 'AssistantMessage' instance.
+     * @param template Template to use for this section.
+     * @param tokens Optional. Sizing strategy for this section. Defaults to `auto`.
+     * @param assistantPrefix Optional. Prefix to use for assistant messages when rendering as text. Defaults to `assistant: `.
```

Comment:
This is for logging, right? So we don't need to take tokens into consideration? If we do, would something shorter like AI: be better?

I'm inclined towards capitalized Assistant. Thoughts?